sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/cosmos/AzureCosmosAutoConfiguration.java

```diff
@@ -59,7 +59,7 @@ public CosmosClient azureCosmosClient(CosmosClientBuilder builder) {
      */
     @Bean
     @ConditionalOnMissingBean
-    // TODO (xiada): spring data cosmos also defines a CosmosAsyncClient
+    // TODO: spring data cosmos also defines a CosmosAsyncClient
```

Comment:
I don't think this should be changed, it is just false alarm from the spell checking tool, the correct way is fixing it there

Reply:
Fixes in https://github.com/Azure/azure-sdk-for-java/pull/32844